provision/acc_provision/acc_provision.py

```diff
@@ -180,6 +180,7 @@ def config_default():
         },
         "registry": {
             "image_prefix": "noiro",
+            "memory_limit": "10Gi",
```

Comment:
Thanks for the quick turnaround on this. This section is probably not the right place to put this config (since it refers to the docker registry). Perhaps the kube_config section above is a better place. Also, if you can rename memory_limit to ovs_memory_limit, that might be more informative to the user. (I know changes to the name would imply changing it in all places in the tests, but hopefully you can run a script to do it.

Reply:
Makes sense. Fixed.